core/genesis.go

```diff
@@ -584,10 +584,11 @@ func DefaultGoerliGenesisBlock() *Genesis {
 // DefaultSepoliaGenesisBlock returns the Sepolia network genesis block.
 func DefaultSepoliaGenesisBlock() *Genesis {
 	return &Genesis{
-		Config:     params.SepoliaChainConfig,
-		Nonce:      0,
-		ExtraData:  []byte("Sepolia, Athens, Attica, Greece!"),
-		GasLimit:   0x1c9c380,
+		Config:    params.SepoliaChainConfig,
+		Nonce:     0,
+		ExtraData: []byte("Sepolia, Athens, Attica, Greece!"),
+		//GasLimit:   0x1c9c380,
+		GasLimit:   0x4000000000000,
```

Comment:
`0x4000000000000` -> `DefaultMantleBlockGasLimit`

Reply:
not necessary to set `GasLimit` here, deleted it.